NickvisionTagger.GNOME/Views/MainWindow.cs

```diff
@@ -703,7 +663,7 @@ private void InsertAlbumArt(AlbumArtType type)
     /// Occurs when the export album art action is triggered
     /// </summary>
     /// <param name="type">AlbumArtType</param>
-    private void ExportAlbumArt(AlbumArtType type)
+    private async Task ExportAlbumArt(AlbumArtType type)
```

Comment:
`ExportAlbumArtAsync`
*Whatever actions calls this method make sure you add `await` in front*